modules/telco-core-common-baseline-model.adoc

```diff
@@ -46,3 +54,4 @@ Telco CNFs can use Service Mesh.
 All telco core clusters require a Service Mesh implementation.
 The choice of implementation and configuration is outside the scope of this specification.
 
+
```

Comment:
The Service Mesh section above this line was updated but has not made it here yet